src/NNlib.jl

```diff
@@ -61,6 +61,9 @@ export conv, conv!, ∇conv_data, ∇conv_data!, ∇conv_filter,
 include("conv_bias_act.jl")
 export conv_bias_act, conv_bias_act!
 
+include("fold.jl")
+export unfold, unfold!, fold, fold!
```

Comment:
I'm a little worried these names may be too common to export. `scatter` collided with every plotting library... 

It's not working for me right now but https://juliahub.com may be able to tell us.

Reply:
https://juliahub.com/ui/Search?q=fold&type=symbols&t=function&u=define

Reply:
possible name confusion with Base too. Given these functions are somewhat domain-specific, I agree it would be better to keep them unexported.

Reply:
No problem, makes sense. That juliahub tool is very useful, thanks for showing.